apisix/core/ctx.lua

```diff
@@ -274,6 +274,15 @@ do
                     end
                 end
 
+            elseif core_str.has_prefix(key, "uri_arg_") then
+                if t._ctx.curr_req_matched then
+                    local arg_key = (sub_str(key, 9))
+                    val = t._ctx.curr_req_matched[arg_key]
+                    if val == nil then
+                        log.warn("failed to fetch uri argument: ", arg_key)
```

Comment:
I think we don't need the warn log like the other branches

Reply:
This was inspired by https://github.com/apache/apisix/blob/master/apisix/core/ctx.lua#L245 
Not sure why this should not write a warning.

Reply:
The reason why the place you pointed add a warn log is due to the the error returned needs to be handled, but you don't have an error to handle, right?
![image](https://github.com/apache/apisix/assets/9354193/547bd062-300c-4be0-9ffd-ce0429edbc98)

Reply:
I have remove the log message.